### PR TITLE
[HOTFIX] Adjust doc file regexp

### PR DESCRIPTION
**Pull request by iwai-wovn**
### Purpose/goal of PR

Document files (e.g. doc/xls) are not eligible for translation.

### CODE REVIEW

File: src/wovnio/wovnphp/Utils.php [hunk 1]
```diff
@@ -169,4 +175,9 @@ private static function getPath($uri)
         $uri_path = preg_replace("/\?(.)*$/", "", $uri_path);
```
Comment (bstewart00) [inline src/wovnio/wovnphp/Utils.php:175]: Could replace this with removeQueryAndHash. I think it makes sense to remove hash as well.

**Comment by Soichiro725** [comment]
@bstewart00 
Do you think this PR is needed? As you suggested, this PR will be replaced by `Content-Type` way.

**Comment by bstewart00** [comment]
It's a tiny PR so I think it's fine to do this first even if we replace it later.

**Comment by Soichiro725** [comment]
> It's a tiny PR so I think it's fine to do this first even if we replace it later.

OK

File: src/wovnio/wovnphp/Utils.php (hunk 1 above)
Comment (iwai-wovn) [reply src/wovnio/wovnphp/Utils.php]: Thx! change this line.

**Pull request merged**